src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.Development.json

```diff
@@ -1,7 +1,7 @@
 {
   "ConnectionStrings": {
     "StatisticsDb": "Server=db;Database=master;User=SA;Password=Your_Password123;",
-    "PublicStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;"
+    "PublicStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"
```

Comment:
Is there any other environment config needed putting in ahead of rolling this PR out onto other environments?

Reply:
I'll need to check the connection string on the enviroment, it should already be there as the current proof of concept is working.

Reply:
Yep all good, the one above is just for local dev